compose/cli/main.py

```diff
@@ -54,7 +54,7 @@ def main():
     try:
         command = TopLevelCommand()
         command.sys_dispatch()
-    except KeyboardInterrupt:
+    except (KeyboardInterrupt, signals.ShutdownException):
```

Comment:
This is added for the `logs` case, where we haven't setup any special try/except for this exception, so we can default to just treating it like a SIGINT.